0-bootstrap/modules/jenkins-agent/main.tf

```diff
@@ -0,0 +1,260 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  cicd_project_name           = format("%s-%s", var.project_prefix, "cicd")
+  impersonation_enabled_count = var.sa_enable_impersonation ? 1 : 0
+  activate_apis               = distinct(var.activate_apis)
+  jenkins_gce_fw_tags         = ["ssh-jenkins-agent"]
+}
+
+resource "random_id" "suffix" {
+  byte_length = 2
+}
+
+/******************************************
+  CICD project
+*******************************************/
+module "cicd_project" {
+  source                      = "terraform-google-modules/project-factory/google"
+  version                     = "~> 7.0" // TODO(caleonardo): use latest current versions so that we are up to date
+  name                        = local.cicd_project_name
+  random_project_id           = true
+  disable_services_on_destroy = false
+  folder_id                   = var.folder_id
+  org_id                      = var.org_id
+  billing_account             = var.billing_account
+  activate_apis               = local.activate_apis
+  labels                      = var.project_labels
+}
+
+/******************************************
+  Jenkins Agent GCE instance
+*******************************************/
+resource "google_service_account" "jenkins_agent_gce_sa" {
+  project      = module.cicd_project.project_id
+  account_id   = format("%s-%s", var.service_account_prefix, var.jenkins_agent_sa_email)
+  display_name = "Jenkins Agent (GCE instance) custom Service Account"
+}
+
+data "template_file" "jenkins_agent_gce_startup_script" {
+  // Add Cloud NAT for the Agent to reach internet and download updates and necessary binaries
+  // not needed  if user has a golden image with all necessary packages.
+  template = file("${path.module}/files/jenkins_gce_startup_script.sh")
+  vars = {
+    tpl_TERRAFORM_DIR     = "/opt/builder/terraform/"
```

Comment:
nit:fmt

Reply:
@caleonardo looks like this PR still has it.  8dca4f1 doesn't seem to have any changes for `0-bootstrap/modules/jenkins-agent/main.tf` introduced in 2e1eddd I think.

Reply:
Found the offending file! - I somehow duplicated a folder. It is fixed now.

Thank you!